src/main/resources/templates/senderTemplate.hbs

```diff
@@ -1,27 +1,37 @@
+{{!-- Helper function to handle status codes --}}
+{{#*inline "statusExit"}}
+    {{#when Response.statuscode 'eq' code}}
+        <Exit path="{{path}}" state="{{state}}" code="{{code}}" {{#when Response.isEmpty 'eq' "true"}}empty="true"{{/when}} />
+    {{/when}}
+{{/inline}}
+
 <Configuration
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="../FrankConfig.xsd"
 >
-    <Adapter name="{{name}}" description="{{description}}">
 
-        <!-- <Enter your code here> -->
+    <!-- <Enter your code here> -->
 
+    <Adapter name="{{name}}" description="{{description}}">
         <Pipeline>
             <Exits>
```

Comment:
Kan dit exits blok in een aparte include oid? Dit is volgens mij precies hetzelfde als hierboven in receivertemplate. Daarmee komt inlineStatusExit ook gelijk op een logische plek uit.